pkg/readiness/readiness.go

```diff
@@ -26,7 +27,21 @@ import (
 	"github.com/projectcalico/node/pkg/readiness/bird"
 )
 
-const felixReadinessEp = "http://localhost:9099/readiness"
+var felixReadinessEp string
+
+func init() {
+	felixPort := os.Getenv("FELIX_HEALTHPORT")
```

Comment:
I think we need to support more than just the environment variable, since Felix configuration parameters can also be configured via config file or datastore config. There might be some Felix code we can import / copy here to make sure we're using the same order of operations?

CC @fasaxc

Reply:
Might be more trouble than its worth.  Env vars override the other config mechanisms so every env var in our manifest prevents other config mechanisms from having an effect.  The manifest seems like a sensible place to co-configure felix and the liveness checker.

Reply:
Yeah, could be. Just feels like something that will come back to bite us. I'd prefer not to leave that bit of debt lying around.